package.json

```diff
@@ -18,7 +18,7 @@
     "start:visual:storybook": "(adb reverse tcp:7007 tcp:7007 || true) && start-storybook",
     "start:types": "tsc --w",
     "start:unit": "npm run test -- --watch",
-    "test": "jest .",
+    "test": "./node_modules/jest/bin/jest.js .",
```

Comment:
Changed to this form due to a suggestion from https://github.com/expo/expo/issues/9129